playground/pages/index.vue

```diff
@@ -75,7 +75,8 @@ definePageMeta({
       <NuxtLink :to="localePath('/')">Home</NuxtLink> | <NuxtLink :to="localePath({ name: 'about' })">About</NuxtLink> |
       <NuxtLink :to="localePath({ name: 'blog' })">Blog</NuxtLink> |
       <NuxtLink :to="localePath({ name: 'category-id', params: { id: 'foo' } })">Category</NuxtLink> |
-      <NuxtLink :to="localePath({ name: 'history' })">History</NuxtLink>
+      <NuxtLinkLocale :to="{ name: 'history' }" class="history-link">History</NuxtLinkLocale> |
+      <NuxtLinkLocale :to="'/'" lang="ja" activeClass="link-active">Home (Japanese)</NuxtLinkLocale>
```

Comment:
I think it should named `locale` prop, not `lang` prop, because in the i18n context, we are mostly naming `locale`, not `lang`.

The ECMAScript Internationalization API also named with locale. The first argument of the `Intl.DateTimeFormat` constructor is called as `locales`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat

Reply:
That makes sense and is consistent with the rest of the module, I've changed the prop to `locale`.